Changes

```diff
@@ -5,6 +5,9 @@ Working version
 
 ### Language features:
 
+- GPR#1066: Allow symbol characters in delimiters for quoted
+  strings. (Matthew Wahab)
```

Comment:
Please follow the style of other entries - there should be a newline before the left parenthesis.